src/cppcache/src/CacheXmlParser.cpp

```diff
@@ -1165,7 +1166,7 @@ void CacheXmlParser::startPersistenceManager(const xmlChar** atts) {
         throw CacheXmlException(s.c_str());
       }
 
-      ACE_OS::strncpy(libraryFunctionName, (char*)atts[i], len);
+      std::strncpy(libraryFunctionName, (char*)atts[i], len);
```

Comment:
Okay, so `xmlChar` is an `unsigned char` and `strncpy ` requires a `char`, hence a c-style cast/hack here.  That seems broken.  A fix may be using `std::memcpy`.  However, other calls, like `strlen` seem fundamentally broken.  I think fixing these calls may be out of scope for this PR.